plugins/by-name/neorg/default.nix

```diff
@@ -20,6 +20,14 @@ with lib;
       ];
     };
 
+    neorgTelescopePackage = lib.mkPackageOption pkgs "neorg-telescope" {
+      nullable = true;
+      default = [
+        "vimPlugins"
+        "neorg-telescope"
+      ];
+    };
```

Comment:
The alternate syntax can be used if you prefer:

```nix
neorgTelescopePackage = lib.mkPackageOption pkgs [
  "vimPlugins"
  "neorg-telescope"
] { nullable = true; };
```

The tail of the list will be used as the "name".